samples/line-time-point-data.html

```diff
@@ -46,8 +46,15 @@
         };
 
         var config = {
-            type: 'line',
+            type: 'line',            
             data: {
+                labels: [
+                  "12/31/2014 06:00",
+                  "01/04/2015 13:00",
+                  "01/07/2015 01:15",
+                  "01/15/2015 01:15"
+                ],
+
```

Comment:
This is actually unnecessary now.  The example is meant to illustrate the capability of the datapoints using `x` and `y` properties instead of the common labels array.